src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static User stepan;
+    private RegistrationServiceImpl registerService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        stepan = new User();
+        stepan.setAge(18);
+        stepan.setId(12L);
+        stepan.setLogin("Stepan");
+        stepan.setPassword("test123");
+    }
+
+    @Test
+    void checkRegisterNull() {
+        try {
+            registerService.register(null);
+        } catch (RuntimeException e) {
+            Assertions.assertEquals(e.getMessage(), "User cannot be null");
+        }

```

Comment:
let's use this instead of try-catch https://howtodoinjava.com/junit5/expected-exception-example/
the same for other try-catches